docker-compose.yml

```diff
@@ -1,349 +1,383 @@
-  environment:
-#JFrog Artifactory server (required) - Can provide multiple 
-  - ARTIFACTORY_URL=https://www.jfrog.com
-# Matches maven artifacts of the form [org]/[module]/[version]/[module]-[version]([-classifier])(.[ext])
-  - ARTIFACTORY_PATTERN0=(?<group>.+)/(?<module>[^/]+)/(?<version>[^/]+)/(?<artifact>\\k<module>)-\\k<version>(-(?<classifier>[^\\.]+))?(\\.(?<ext>.+))?
-# Matches ivy files of the form [org]/[module]/[revision]/ivy-[revision](-[classifier]).xml 
-  - ARTIFACTORY_PATTERN1=(?<group>.+)/(?<module>[^/]+)/(?<version>[^/]+)/(?<artifact>ivy)-\\k<version>(-(?<classifier>[^\\.]+))?\\.(?<ext>xml)
-# Matches ivy artifact files of the form [org]/[module]/[revision]/[type]/[artifact]-[revision](-[classifier])(.[ext])
-  - ARTIFACTORY_PATTERN2=(?<group>.+)/(?<module>[^/]+)/(?<version>[^/]+)/(?<type>[^/]+)/(?<artifact>[^\\.-/]+)-\\k<version>(-(?<classifier>[^\\.]+))?(\\.(?<ext>.+))?
-
-  
-# Artifactory user name (required)
-#  - ARTIFACTORY_USERNAME=bobama
-# Artifactory API key (required)
-#  - ARTIFACTORY_API_KEY=s3cr3t
-#  - ARTIFACTORY_CRON=0 0/1 * * * *
-# The repo to collect artifacts from (required) - Can provide multiple (comma separated for each server)
-#  - ARTIFACTORY_REPO=prerelease,release
-# Artifactory REST endpoint
-#  - ARTIFACTORY_ENDPOINT=artifactory/
-
-hygieia-appdynamics-collector:
-  image: hygieia-appdynamics-collector:latest
-  container_name: hygieia-appdynamics
-  volumes:
-  - ./logs:/hygieia/logs
-  links:
-  - mongodb:mongo
-  - hygieia-api
-# place any values you want to change in docker-compose.override.yml
-#  environment:
-#  - APPDYNAMICS_INSTANCE_URL=http://appdynamics
-#  - APPDYNAMICS_USERNAME=username
-#  - APPDYNAMICS_PASSWORD=password
-#  - APPDYNAMICS_DASHBOARD_URL=http://appdynamics/controller/#/location=APP_DASHBOARD&timeRange=last_15_minutes.BEFORE_NOW.-1.-1.15&application=%s&dashboardMode=force
-#  - APPDYNAMICS_CRON='-1 * * * * *'
+version: '2'
+volumes:
+  data: {}
+services:
+  mongodb:
+    image: mongo:latest
+    container_name: mongodb
+    #command: mongod --smallfiles
+    ports:
+     - "27017:27017"
+    volumes:
+     - data:/data/db:rw
+    environment:
+     - MONGODB_USERNAME=db
+     - MONGODB_DATABASE=dashboard
+     - MONGODB_PASSWORD=dashboard
+    volume_driver: local
+  db-setup:
+    build: db
+    depends_on: [mongodb]
+    container_name: db_doer
+    links:
+      - mongodb
+    #links now use hostnames instead of ENV Vars
+    environment:
+      - MONGO_STARTED=yes
+  hygieia-api:
+    build: api/docker
+    image: library/hygieia-api:latest
+    container_name: hygieia-api
+    ports:
+      - "8080:8080"
+    volumes:
+    - ./logs:/hygieia/logs
+    environment:
+      - MONGO_STARTED=yes
+      - jasypt.encryptor.password=hygieiasecret
+      - SPRING_DATA_MONGODB_DATABASE=dashboard
+      - SPRING_DATA_MONGODB_HOST=mongodb
+      - SPRING_DATA_MONGODB_PORT=27017
+      - SPRING_DATA_MONGODB_USERNAME=db
+      - SPRING_DATA_MONGODB_PASSWORD=dashboard
```

Comment:
This should be `- SPRING_DATA_MONGODB_PASSWORD=dbpass` in order to match the values found in [db-setup.js](https://github.com/capitalone/Hygieia/blob/master/db/db-setup.js).

Reply:
On second thought both should be changed to reflect the documentation